### PR TITLE
Add more headings to admin settings page - Fixes #181, fixes #182

### LINKED ISSUE #181: "Use proxy" is not a per-activity-instance setting
Hi all,

looking at /admin/settings.php?section=modsettingzoom, I would like to mention that the setting "Use proxy" is not a per-activity-instance setting. It should be moved out of the "Default Zoom settings" fieldset and be moved to the top fieldset on this page where the admin sets the general settings of the plugin.

Cheers,
Alex

### LINKED ISSUE #182: "Display passcode" is not a per-activity-instance setting
Hi all,

looking at /admin/settings.php?section=modsettingzoom, I would like to mention that the setting "Display passcode" is not a per-activity-instance setting. It should be moved out of the "Default Zoom settings" fieldset and be moved to the top fieldset on this page where the admin sets the general settings of the plugin.

Cheers,
Alex

### PR DESCRIPTION
**Pull request by abias**
This patch adds some headings to the admin settings page to cluster the various settings for better understandability.
Additionally, it fixes a small flaw with the identifier of the default settings heading.

### CODE REVIEW

File: settings.php [hunk 1]
```diff
@@ -63,15 +68,15 @@
...
-    $jointimeselect = array();
-    foreach ($jointimechoices as $minutes) {
-        $jointimeselect[$minutes] = $minutes . ' ' . get_string('mins');
-    }
-    $firstabletojoin = new admin_setting_configselect('mod_zoom/firstabletojoin',
-            get_string('firstjoin', 'mod_zoom'), get_string('firstjoin_desc', 'mod_zoom'),
-            15, $jointimeselect);
-    $settings->add($firstabletojoin);
+    $proxyhost = new admin_setting_configtext('mod_zoom/proxyhost',
+            get_string('option_proxyhost', 'zoom'),
```
Comment (jrchamp) [inline settings.php:72]:
```suggestion
            get_string('option_proxyhost', 'mod_zoom'),
```

File: settings.php [hunk 2]
```diff
@@ -88,13 +93,33 @@
...
+        $jointimeselect[$minutes] = $minutes . ' ' . get_string('mins');
+    }
+    $firstabletojoin = new admin_setting_configselect('mod_zoom/firstabletojoin',
+            get_string('firstjoin', 'mod_zoom'), get_string('firstjoin_desc', 'mod_zoom'),
+            15, $jointimeselect);
+    $settings->add($firstabletojoin);
+
+    $displaypassword = new admin_setting_configcheckbox('mod_zoom/displaypassword',
+            get_string('displaypassword', 'zoom'),
+            get_string('displaypassword_help', 'zoom'), 0, 1, 0);
```
Comment (jrchamp) [inline settings.php:113]:
```suggestion
            get_string('displaypassword', 'mod_zoom'),
            get_string('displaypassword_help', 'mod_zoom'), 0, 1, 0);
```

**Review by jrchamp — COMMENTED** [review]
Noticed a couple of text-string namespace issues that predate these changes.

**Comment by abias** [comment]
Thanks for your review, @jrchamp .

As far as I see, both ways of calling get_string() for activity plugins work, but I fully agree that all calls should be aligned.

I have fixed this now in a separate commit and have force-pushed this PR branch.

PS: If you are in aligning things currently, you might want to look at #180 as well.

**Pull request merged**